### PR TITLE
PLAT-144650: Fix scroller moves abnormally in RTL locales

### PR DESCRIPTION
**Pull request by jeonghee27**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Scroll position moves abnormally in RTL horizontal scroller.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Apply negative coordinate for calculatePositionOnFocus in `Scroller`. 

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-144650

### Comments
Enact-DCO-1.0-Signed-off-by: Jeonghee Ahn (jeonghee27.ahn@lge.com)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ The following is a curated list of changes in the Enact sandstone module, newest
 - `sandstone/MediaPlayer` to show `MediaControls` via wheel properly when isomorphic build
 - `sandstone/Popup` to have proper focus when opening with `noAnimation` is `true`
 - `sandstone/PopupTabLayout` to move focus via 5-way left in the header
+- `sandstone/Scroller` to scroll correctly on Chrome 85 or higher in RTL locales
```
Comment (MikyungKim) [inline CHANGELOG.md:24]: Does this make sense?
```suggestion
- `sandstone/Scroller` to scroll correctly on Chrome 85 or higher in RTL locales via 5way
```

File: Scroller/useThemeScroller.js [hunk 2]
```diff
@@ -271,13 +272,14 @@ const useSpottable = (props, instances) => {
 
 		const
 			{rtl} = props,
+			negativeCoordinate = rtl && (platform.ios || platform.safari || platform.chrome >= 85) ? -1 : 1,
```
Comment (MikyungKim) [inline Scroller/useThemeScroller.js:275]:
```suggestion
			negativeCoordinateCoefficient = rtl && (platform.ios || platform.safari || platform.chrome >= 85) ? -1 : 1,
```

File: Scroller/useThemeScroller.js [hunk 3]
```diff
@@ -271,13 +272,15 @@ const useSpottable = (props, instances) => {
 
 		const
 			{rtl} = props,
+			// For Chrome 85+ or Safari that use negative coordinate sysmte for RTL
```
Comment (MikyungKim) [inline Scroller/useThemeScroller.js:275]:
```suggestion
			// For Chrome 85+ or Safari that use negative coordinate system for RTL
```

File: Scroller/useThemeScroller.js [hunk 4]
```diff
@@ -271,13 +272,15 @@ const useSpottable = (props, instances) => {
 
 		const
 			{rtl} = props,
+			// For Chrome 85+ or Safari that use negative coordinate system for RTL
+			coordinateoCoefficient = rtl && (platform.ios || platform.safari || platform.chrome >= 85) ? -1 : 1,
```
Comment (MikyungKim) [inline Scroller/useThemeScroller.js:276]:
```suggestion
			coordinateCoefficient = rtl && (platform.ios || platform.safari || platform.chrome >= 85) ? -1 : 1,
```

**Review by MikyungKim — APPROVED** [review]
:lgtm:

**Comment by 0x64** [comment]
Note: we need to cover other browsers later.

**Pull request merged**